src/style.css

```diff
@@ -1 +1,79 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  background: #e5e5e5;
+  height: 100%;
+}
+
+.header {
+  --height-heder: 60px;
```

Comment:
don't harcode height, use padding instead

Reply:
did not understand the comment

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/2635#discussion_r1097568656
use paddings for links, instead of hardcoding header height